code_generator/Templates/UnityCommandPayloadGenerator.tt

```diff
@@ -23,17 +23,21 @@ namespace <#= qualifiedNamespace #>
 
                 internal Translation Translation { get; }
 
+                public string CallerWorkerId;
```

Comment:
should we also make it readonly or do we want to change the `CallerWorkerId` after creating the request.

Reply:
Possibly, might be worth making everything else `readonly` then

Reply:
Nvm just noticed the `{ get; }` on everything else